staff/aperacaula/cv/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Alex Peracaula</title>
+</head>
+
+<body>
+    <h1>CV Àlex Peracaula</h1>
+    <h3>mail: aperacaularuiz@gmail.com
+        <br> tfn: 630075725</h3>
+    <h2>Experiencia Laboral</h2>
+    <ul>
+        <li>Feb 17-Aug 17: Intern in Strategy Planning Analysis in Principle Power Inc.</li>
+        <li>Feb 15-Aug 15: Scholar Researcher in National Fuel Cell Research Center</li>
+
+    </ul>
+    <h2>Educación</h2>
+    <ul>
+        <li>Double Master Degree Renewable Energy Engineering by Innoenergy
```

Comment:
WARN cv is mainly in spanish, why this in english?

Reply:
I chose to put it in the language of the Master's Degree, to give info about it, but perhaps you're right!

Reply:
DONE 👌🏻